src/animation/shine.js

```diff
@@ -27,9 +28,12 @@ const Shine = ({ children }) => {
       Animated.timing(animation, {
         toValue: END_VALUE,
         duration: DURATION,
+        useNativeDriver,
       }),
-    ]).start(() => {
-      start();
+    ]).start((e) => {
+      if (e.finished) {
```

Comment:
Well, I don't know if it's a way to solve the problem:

When calling `start`, we start the animation. I think the callback is immediately called. What is it's not finished yet? Does it mean that we would run the animation only once?

Thanks for your help 👍

Reply:
I also see this issue and testing (e.finished) does fix the memory leak we are seeing.

The callback to start is called when the animation is complete and not immediately, so we must test if it finished correctly or was terminated for some reason, otherwise any cancelled or stopped animation will just be started again.

Reply:
I'm not sure the OP will continue this pr 😅 

You'd like to take the point on a separate PR? Or would you like me to dot it? 😊

Reply:
But useNativeDriver is not needed and crashes for me as per you comment marginLeft is not supported